go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/processout/braintree-go
 
-go 1.21.1
+go 1.18
 
-require github.com/stretchr/testify v1.8.4
+require github.com/stretchr/testify v1.8.2
```

Comment:
The same goes here - Tessel9 uses this version of testify and we won't need to upgrade it either.